pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApi2Test.java

```diff
@@ -1648,6 +1648,11 @@ public void testDeleteNamespaceWithTopicPolicies() throws Exception {
 
         // create namespace2
         String namespace = tenant + "/test-ns2";
+        admin.namespaces().createNamespace(namespace, Set.of("test"));
+        admin.topics().createNonPartitionedTopic(namespace + "/tobedeleted");
+        // verify namespace can be deleted even without topic policy events
```

Comment:
I'm not sure why the namespace couldn't be deleted when there is no topic policy events.

Reply:
it's the scope of this pull 😁 
When deleting namespace with force=true, you first delete the topics in it. For each topics deleted, a new event is generated. However if the __change_events doesn't exist yet, it's auto-created. But the creation fails since the namespace is being deleted.

Reply:
btw I've added a more specific unit test case